### PR TITLE
Move and update the OpenStack page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated the content with the latest copy doc. Moved from `/cloud/openstack` to `openstack`. Updated all links to it across the site.

Removed the `/cloud/openstack/autopilot` page and removed all links to it.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Check the copy matches the [copy doc](https://docs.google.com/document/d/12z6QEGpqMVbNPJOd5aUfn0VnXQ0Sa5YriEBLA7VGNwI/edit)
- Check the menu is correct

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2186
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2221

### CODE REVIEW

File: templates/openstack/index.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      <h2>Three ways to get OpenStack on Ubuntu</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-4 p-card">
+        <h3 class="p-card__title">Fully managed OpenStack cloud</h3>
+        <p class="p-card__content">Canonical will design, build and operate a production OpenStack cloud on your premises from $15 per server per day. We&rsquo;ll train and transfer control to your team on request.</p>
+        <p class="p-card__content">This is the recommended way to get to production fast, enabling you to ramp your workloads efficiently and focus on cloud consumption not infrastructure.</p>
+        <p class="p-card__content"><a href="/cloud/managed-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Openstack', 'eventLabel' : 'Get your BootStack engagement', 'eventValue' : undefined });">Get your BootStack engagement&nbsp;&rsaquo;</a></p>
```
Comment (grahambancroft) [inline templates/openstack/index.html]: Different text and link in the copy doc

File: templates/openstack/index.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      </div>
+      <div class="col-4 p-card">
+        <h3 class="p-card__title">OpenStack delivered to you in 3 weeks</h3>
+        <p class="p-card__content">Tailor your reference  OpenStack cloud to your requirements and then our engineers will build your, working alongside your team.</p>
+        <p class="p-card__content">Canonical offers training and long-term enterprise support plus optional consulting services for networking, storage, monitoring and cloud architecture.</p>
+        <p class="p-card__content"><a href="/cloud/foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Openstack', 'eventLabel' : 'Learn more about Foundation Cloud Build', 'eventValue' : undefined });">Learn more about Foundation Cloud Build&nbsp;&rsaquo;</a></p>
```
Comment (grahambancroft) [inline templates/openstack/index.html]: Different link in the copy doc

File: templates/openstack/index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      </div>
+      <div class="col-4 p-card">
+        <h3 class="p-card__title">Install manually &ndash; single or multi-node</h3>
+        <p class="p-card__content">Use Canonical&rsquo;s conjure-up to deploy a multi-node OpenStack. With LXD containers, it all fits on your laptop, in VMware or even a public cloud. Ideal for first-time users or developers who want to study the components of OpenStack in action. Use it with MAAS (&lsquo;Metal as a Service&rsquo;) to deploy across many servers and scale it to many racks.</p>
+        <p class="p-card__content"><a href="/download/cloud/conjure-up" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Cloud openstack', 'eventLabel' : 'Try conjure-up', 'eventValue' : undefined });">Install OpenStack manually&nbsp;&rsaquo;</a></p>
```
Comment (grahambancroft) [inline templates/openstack/index.html]: Different link in the copy doc

File: templates/openstack/index.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
```
Comment (grahambancroft) [inline templates/openstack/index.html]: Ebay quote is missing

File: templates/openstack/index.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  </div>
+  <div class="row">
+    <div class="col-7">
+      <p>With our Foundation Cloud Build service you can get a full production OpenStack cloud up and running in 3 weeks using the reference architecture. With model driven operations and upgrades to future OpenStack releases guaranteed, the OpenStack cloud from Canonical is the fast track to efficient cloud success. Optional consulting available to tailor the architecture to your specific requirements.</p>
+      <ul class="p-list is-split">
+        <li class="p-list__item is-ticked">SLAs and support services</li>
+        <li class="p-list__item is-ticked">On-site engineering and support</li>
+        <li class="p-list__item is-ticked">Reference architecture available</li>
+        <li class="p-list__item is-ticked">OpenStack feature development as needed</li>
+        <li class="p-list__item is-ticked">Customized architecture service placement</li>
```
Comment (grahambancroft) [inline templates/openstack/index.html]: Wrong spelling of customized, I've corrected the doc

File: templates/openstack/index.html (hunk 4 above)
Comment (anthonydillon) [reply templates/openstack/index.html]: This is being removed for now.

File: templates/openstack/index.html (hunk 5 above)
Comment (anthonydillon) [reply templates/openstack/index.html]: Thanks

File: templates/openstack/index.html (hunk 3 above)
Comment (anthonydillon) [reply templates/openstack/index.html]: This has been updated in the copy doc.

File: templates/openstack/index.html (hunk 2 above)
Comment (anthonydillon) [reply templates/openstack/index.html]: This has been updated in the copy doc.

File: templates/openstack/index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/openstack/index.html]: Done, thanks.

File: redirects.yaml [hunk 6]
```diff
@@ -982,7 +983,6 @@ node/2316/?: "https://insights.ubuntu.com/"
 occs/?: "http://partners.ubuntu.com/ibm"
 ods/?: "https://insights.ubuntu.com/2014/11/03/extreme-openstack/?utm_source=ubunteu&utm_medium=url_shortner&utm_term=ods2014paris&utm_campaign=shortner"
 oil/?: "http://partners.ubuntu.com/programmes/openstack"
-openstack/?: "/cloud/openstack"
```
Comment (WillMoggridge) [inline redirects.yaml]: I imagine we would want to add this in reverse, to redirect `/cloud/openstack` to `/openstack`

Comment (anthonydillon) [reply redirects.yaml]: This exists on line 73

Comment (WillMoggridge) [reply redirects.yaml]: Ah, it isn't with the other alphabetical lines. 🤔

**Pull request merged**